MoreLinq/NoConflict.ToDataTable.cs

```diff
@@ -0,0 +1,89 @@
+namespace MoreLinq.NoConflict
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq.Expressions;
+
+    /// <summary><c>ToDataTable</c> extension.</summary>
+    public static partial class ToDataTableExtension
+    {
+        /// <summary>
+        /// Appends elements in the sequence as rows of a given <see cref = "DataTable"/> 
```

Comment:
The spacing issue in XML comments should be gone now with 11a0f47cba0ae125a2947e3d54212893499a9fb3. I've removed the call to `NormalizeWhitespace` and added formatting manually, which I was hoping to avoid but oh well.